include/TGUI/Widgets/ChildWindow.hpp

```diff
@@ -399,8 +503,14 @@ namespace tgui
         Label          m_titleText;
         sf::Vector2f   m_draggingPosition;
         TitleAlignment m_titleAlignment = TitleAlignment::Center;
+        TitleButtons   m_titleButtons = TitleButtons::Close;
+        sf::String     m_closeButtonText = "x";
+        sf::String     m_minimizeButtonText = "-";
+        sf::String     m_maximizeButtonText = "+";
```

Comment:
Do the button texts even need to be member variables of the class? The buttons have getters and setters so perhaps this is redundant?